docs/release-notes/Percona-Everest-1.7.0-(2025-05-29).md

```diff
@@ -0,0 +1,218 @@
+# 📦 What's new in Percona Everest 1.7.0
+
+➡️ **New to Percona Everest?** Get started with our [Quickstart Guide](https://docs.percona.com/everest/quick-install.html).
+
+
+??? info "🔑 Expand to unleash the key updates"
+
+    |**#**|**Release summary**|**Description**|
+    |---------|---------------------|---------|
+    | **1.**|[GKE Autopilot clusters](https://docs.percona.com/everest/release-notes/Percona-Everest-1.6.0-%282025-04-16%29.html#__tabbed_1_1)|Deploy Percona Everest on Google Kubernetes Engine (GKE) Autopilot clusters|
+    | **2.**|[Pod Scheduling policies](https://docs.percona.com/everest/release-notes/Percona-Everest-1.6.0-%282025-04-16%29.html#__tabbed_1_1)|Pod Scheduling for optimized Kubernetes scheduling¶|
+    | **3.**|[TLS support](https://docs.percona.com/everest/release-notes/Percona-Everest-1.6.0-%282025-04-16%29.html#__tabbed_1_2)|Improved Security with TLS support|
+    | **4.**|[Operator Upgrades](https://docs.percona.com/everest/release-notes/Percona-Everest-1.6.0-%282025-04-16%29.html#__tabbed_1_3)|Support for Percona XtraBackup Operator 1.17.0|
+    | **5.**|[Google Container Registry (GCR) deprecation](https://docs.percona.com/everest/release-notes/Percona-Everest-1.6.0-%282025-04-16%29.html#google-container-registry-gcr)|Deprecation of GCR starting **May 20, 2025**|
+    | **6.**|[New features](Percona-Everest-1.6.0-%282025-04-16%29.html#new-features)|Check out the new features introduced in Percona Everest 1.7.0|
+    | **7.**|[Improvements](Percona-Everest-1.6.0-%282025-04-16%29.html#improvements)|Discover all the enhancements featured in Percona Everest 1.7.0|
+    | **8.**|[Bugs](Percona-Everest-1.6.0-%282025-04-16%29.html#bugs)|Find out about all the bugs fixed in Percona Everest 1.7.0|
+    | **9.**|[Known limitations](https://docs.percona.com/everest/release-notes/Percona-Everest-1.6.0-%282025-04-16%29.html#known-limitation)|Discover all the known limitations in Percona Everest 1.7.0|
+
+
+## 🌟 Release highlights
+
+=== "☁️ GKE Autopilot clusters"
+
+    ### Deploy Percona Everest on GKE Autopilot
+
+     You can now install Percona Everest on Google Kubernetes Engine (GKE) Autopilot clusters. GKE Autopilot provides a fully managed Kubernetes environment, where Google automatically handles node provisioning, scaling, and security. 
+
+    Here are the benefits of this enhancement:
+
+    - Optimized resource allocation
+    - Streamlined operations
+    - Reduced operational overhead
+    - Simplified cluster management
+    - Automated scaling and built-in security policies—leveraging Percona Everest’s database management capabilities.
+
+
+    📚 Learn more about GKE Autopilot in the official [documentation](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview){:target="_blank"}.
+
+===  "⎈ Pod scheduling policies"
+
+    ### Optimized Pod scheduling policies in Percona Everest
```

Comment:
not sure `optimized` is a good work here, because in fact we introduce this feature for the first time

Reply:
Fixed